pallets/auctions/src/lib.rs

```diff
@@ -91,25 +74,30 @@ pub mod pallet {
 		// This type is needed to convert from Currency to Balance
 		type CurrencyBalance: From<Self::Balance>
 			+ Into<<<Self as crate::Config>::Currency as Currency<<Self as frame_system::Config>::AccountId>>::Balance>;
+
+		/// Limit of auction name length
+		type AuctionsStringLimit: Get<u32>;
+
+		/// Increase end time to avoid sniping
+		type BidAddBlocks: Get<u32>;
+
+		/// Next bid step in percent
+		type BidStepPerc: Get<u32>;
+
+		/// Minimum auction duration
+		type MinAuctionDuration: Get<u32>;
 	}
 
 	#[pallet::storage]
 	#[pallet::getter(fn auctions)]
 	/// Stores on-going and future auctions. Closed auction are removed.
-	// TODO: use single Auction storage using double map (auctionId, type)
-	pub type Auctions<T: Config> = StorageMap<_, Twox64Concat, T::AuctionId, AuctionInfoOf<T>, OptionQuery>;
+	pub type Auctions<T: Config> = StorageMap<_, Twox64Concat, T::AuctionId, Auction<T>, OptionQuery>;
```

Comment:
Does this storage needs to be public ?

Use pub only when it makes sense.

Reply:
All substrate stores i have seen so far are pub but maybe I miss something?

If I make it private, how can I call it under in the `impl<T: Config> Pallet<T>` block?

Reply:
you can do pub(crate) to use in the impl. 

You can leave it pub if you wish. I just feel that a pallet should expose only what needs to be.

Reply:
I agree with you, still learning best practices :) thx!